app/views/announcement_mailer/monthly_warning.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>A scheduled announcement will go out in one week</h1>
+
+<i><%= link_to "Review the announcement in your browser", @announcement %></i>
```

Comment:
prompt them to update the announcement and share about what the org has been working on